src/Client.php

```diff
@@ -78,12 +80,15 @@ class Client
 
     private bool $isRetryFeatureEnabled;
 
+    private int $slowRequestThresholdMs = 500;
```

Comment:
it is set in contructor
```suggestion
    private int $slowRequestThresholdMs;
```